tests/test_standard_tags.py

```diff
@@ -110,11 +111,11 @@ def setup_url_with_sensitive_query_string(self):
         ]
 
     @missing_feature(
-        context.library in ["golang", "nodejs", "php", "ruby"],
+        context.library in ["golang", "nodejs", "php", "ruby", "python", "java"],
         reason="tracer did not yet implemented the new version of query parameters obfuscation regex",
     )
-    @missing_feature(context.library < "java@1.21.0", reason="previous obfuscation regex")
-    @irrelevant(context.library < "python@1.19", reason="python released the new version at 1.19.0")
+    # @missing_feature(context.library < "java@1.21.0", reason="previous obfuscation regex")
+    # @irrelevant(context.library < "python@1.19", reason="python released the new version at 1.19.0")
```

Comment:
If they are no longer needed, it is better to delete

Reply:
fixed in [4430067](https://github.com/DataDog/system-tests/pull/1756/commits/44300678c50738d4409bfac328ab5bb7b43f5363) thanks